### PR TITLE
DS 176 Form element update

### PR DESCRIPTION
**Pull request by remydenton**
## Jira

https://pegadigitalit.atlassian.net/browse/DS-176

## Summary

Updates element demo pattern to include 'for' attribute in label

## Details

The base form element component shouldn't be update as described in the JIRA ticket (and isn't necessary since Drupal takes care of the "for" attribute).  I will go into further detail about why on the ticket.

This is one possible way to ensure that the 'for' attribute is used in our pattern lab demos though.

## How to test

Ensure no regressions in the form element

### CODE REVIEW

File: docs-site/src/pages/pattern-lab/_patterns/50-pages/55-d8-calculator/00-d8-calculator.twig [hunk 1]
```diff
@@ -104,62 +104,36 @@
...
-                    placeholder: "E.g. 2000",
-                    required: true,
-                    type: "number",
-                    id: for_value,
-                    "data-bolt-format-input": "currency-us"
-                  },
-                } only %}
-              {% endset %}
-              {% include "@bolt-components-form/form-element.twig" with {
+              {% include "@pl/10-form-element-demo-input-element.twig" with {
```
Comment (mikemai2awesome) [inline docs-site/src/pages/pattern-lab/_patterns/50-pages/55-d8-calculator/00-d8-calculator.twig:107]: Why did we change this this include? The one before is the correct usage `{% include "@bolt-components-form/form-element.twig"`, while this one is pointing to PL. Drupal devs can't copy and paste this code.

Comment (colbytcook) [reply docs-site/src/pages/pattern-lab/_patterns/50-pages/55-d8-calculator/00-d8-calculator.twig:107]: not sure, but I will check it out

Comment (remydenton) [reply docs-site/src/pages/pattern-lab/_patterns/50-pages/55-d8-calculator/00-d8-calculator.twig:107]: Let's discuss during dev huddle today (as @mikemai2awesome suggested on the JIRA ticket).

**Comment by colbytcook** [comment]
We decided to continue doing the for and id manually to prevent a bunch of work that may not have a large amount of return (Drupal is currently automatically adding these in)

**Pull request closed without merging**